contents/docs/getting-started/next-steps.mdx

```diff
@@ -0,0 +1,107 @@
+
+Read our [web analytics vs product analytics](/docs/web-analytics/web-vs-product-analytics) explainer for more.
+
+## 3. Invite your team
+
+PostHog is way more powerful when you're using it with your teammates.
+
+To invite them, go to your instance homepage, click the **Invite** button, and add their email. 
+
+This will add them to your organization with access to all your projects.
+
+//TKTK update screenshot
+
+![Invite](https://res.cloudinary.com/dmukukwp6/image/upload/v1710055416/posthog.com/contents/images/tutorials/next-steps-after-installing/invite.png)
+
+You can change their access levels in your organization settings, and set some projects to private if required.
+
+You'll likely want to [filter out events from internal users](/tutorials/filter-internal-users) if you do this.
+
+## 4. Watch your first session replay
+
+Watching people using your product is like magic.
+
+Once you've [enabled session replay](/docs/session-replay/installation) in your project, head to the [what to watch tab](https://us.posthog.com/replay/templates) in session replay to watch how users naviagte specific parts of your app.
+
+Warning: watching replays is addictive, but incredibly useful.
+
+## 5. Start sending PostHog data to other tools
+
+As data arrives in PostHog, you can export it _immediately_ to other tools using [realtime destinations](/docs/cdp/destinations/). 
+
+You can use this for things like:
+
+- Conversion tracking
+- Monitoring key events with your chat platform
+- Enriching customer data in your CRM
+- Triggering automations
+
+We have dozens of pre-configured destinations, including [Slack](/docs/cdp/destinations/slack), [Discord](/docs/cdp/destinations/discord), [Hubspot](/docs/cdp/destinations/hubspot), [Intercom](/docs/cdp/destinations/hubspot), and a [webhook destination](/docs/cdp/destinations/webhook) for sending events to any HTTP endpoint.
+
+Head to the [data pipelines](/docs/cdp) docs for more on how you can use PostHog as your customer data platform.
+
+You can also configure [batch exports](/docs/cdp/batch-exports/s3) to a data warehouse.
+
+## 6. Import revenue and customer data into our data warehouse
+
+When you choose PostHog, you also get a built-in data warehouse.
+
+You can then [link your most valuable data](/docs/data-warehouse/join) from sources like your CRM, payment processor, or database, combine it with the event data already in PostHog, and query across all of it.
+
+We currently offer fully-managed import integrations for the likes of [Stripe](/docs/cdp/sources/stripe), [Hubspot](/docs/cdp/sources/hubspot), and [Salesforce](/docs/cdp/sources/salesforce).
+
+We also support important data from most popular data warehouses and databases, such as [Postgres](/docs/cdp/sources/postgres), [BigQuery](/docs/cdp/sources/bigquery), and [Amazon S3](/docs/cdp/sources/s3).
+
+See the full [list of PostHog data sources](/docs/cdp/sources) for more. 
+
+## 7. Figure out what to track in PostHog
```

Comment:
I don't think this is the right framing, maybe we should say "Learn how to win with PostHog" or "Use PostHog to help build a successful product." What to track feels less important than using what you are tracking already to make you successful.